tests/ja4-sv-handshake/test.yaml

```diff
@@ -0,0 +1,14 @@
+requires:
+  min-version: 8
+  features:
+    - HAVE_JA4
```

Comment:
Could we have a test that does not require JA4 ?

Reply:
Could you elaborate for me, please? So you want a test that mirrors the above but just with that removed? So are you expecting it to produce no output? Apologies, I've just not quite understood what you want here.

Reply:
I expected that we can output `tls.client_handshake.exts` even without ja4 enabled...
Is that wrong ?